hoot-core/src/main/cpp/hoot/core/io/OsmGeoJsonReader.cpp

```diff
@@ -433,11 +437,29 @@ void OsmGeoJsonReader::_parseGeoJsonRelation(const string& id, const pt::ptree&
     }
   }
   else if (geo_type == "MultiPoint")
-    _parseMultiPointGeometry(geometry, relation);
+  {
+      if (relation->getType() == "")
+      {
+          relation->setType(MetadataTags::RelationMultiPoint());
+      }
+      _parseMultiPointGeometry(geometry, relation);
+  }
   else if(geo_type == "MultiLineString")
+  {
+      if (relation->getType() == "")
+      {
+          relation->setType(MetadataTags::RelationMultilineString());
```

Comment:
Should this be RelationMulti**L**ineString ??? 

Metadata.h does not think so.

Reply:
I guess that would be more accurate though not a big deal unless you want it to be changed.